src/API/WesternStatesWater.WestDaat.Accessors/EntityFramework/ControlledVocabularyBase.cs

```diff
@@ -0,0 +1,11 @@
+namespace WesternStatesWater.WestDaat.Accessors.EntityFramework;
+
+public abstract class ControlledVocabularyBase
+{
+    public string Name { get; set; }
+    public string Term { get; set; }
+    public string Definition { get; set; }
+    public string State { get; set; }
+    public string SourceVocabularyUri { get; set; }
+    public string WaDEName { get; set; }
```

Comment:
Do we want to carry over the data annotations (`MaxLength` and so on) that were in CoordinateMethod?

Reply:
Never mind, I see it's in the model builder